src/items/items_definitions.hpp

```diff
@@ -606,9 +617,21 @@ enum ItemParseAttributes_t {
 	ITEM_PARSE_PRIMARYTYPE,
 	ITEM_PARSE_USEDBYGUESTS,
 	ITEM_PARSE_SCRIPT,
+	ITEM_PARSE_AUGMENT,
 };
 
 struct ImbuementInfo {
 	Imbuement* imbuement;
 	uint32_t duration = 0;
 };
+
+struct AugmentInfo {
+	AugmentInfo(std::string spellName, AugmentTypes_t type, int32_t value) :
+		spellName(std::move(spellName)), type(type), value(value) { }
+
+	std::string spellName;
+	AugmentTypes_t type;
+	int32_t value;
+};
+
+using Augments = std::vector<std::shared_ptr<AugmentInfo>>;
```

Comment:
I have reservations about using aliases, as they obscure the object types, which can complicate code review. Although aliases can simplify maintenance, they might pose challenges in a large, open-source project like Canary. 
But, I'm not opposing it, it's something particular to each person, as we don't define a model to be followed in the project.

Reply:
I don't like either, I saw some aliases and I thought that was a pattern to follow, reverting.